### ISSUE TITLE
accessibility: <html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by ychoy** [codeforsanjose/census2020#125]
According to this [audit report from Chrome Dev Tools](https://github.com/codeforsanjose/census2020/files/4054578/Census2020-Audit-Report-0122020.pdf), this can be enhanced to improve accessibility:

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. Learn more. https://web.dev/html-has-lang/?utm_source=lighthouse&utm_medium=devtools

Failing Elements
--
`html<html>`
